_posts/2023-05-22-scquickaccounting.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "[CVE-2023-33280] Improper neutralization of multiple SQL parameters in the scquickaccounting module for PrestaShop"
+categories: modules
+author:
+- Store Commander
+- TouchWeb
+- 202 ecommerce
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,scquickaccounting"
+severity: "critical (9.8)"
+---
+
+In the module "SC Quick Accounting" (scquickaccounting), an anonymous user can perform SQL injection. The module have been patched in version 3.7.4.
+
+## Summary
+
+* **CVE ID**: [CVE-2023-33280](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-33280)
+* **Published at**: 2023-
+* **Platform**: PrestaShop
+* **Product**: scquickaccounting
+* **Impacted release**: <= 3.7.3
+* **Product author**: Store Commander
+* **Weakness**: [CWE-89](https://cwe.mitre.org/data/definitions/89.html)
+* **Severity**: critical (9.8)
+
+## Description
+
+In scexportcustomers module for PrestaShop up to 3.7.3, multiple sensitive SQL calls can be executed with a trivial http call and exploited to forge a blind SQL injection.
```

Comment:
Can we add "(fixed in 3.7.4)" to add clarity?

Reply:
yes. Commit done